packages/inference/src/lib/makeRequestOptions.ts

```diff
@@ -1,6 +1,20 @@
 import type { Options, RequestArgs } from "../types";
 
-const HF_INFERENCE_API_BASE_URL = "https://api-inference.huggingface.co/models/";
+const HF_INFERENCE_API_BASE_URL = "https://api-inference.huggingface.co/";
+const HF_INFERENCE_API_PIPELINE_BASE_URL = `${HF_INFERENCE_API_BASE_URL}pipeline/`;
+const HF_INFERENCE_API_MODEL_BASE_URL = `${HF_INFERENCE_API_BASE_URL}models/`;
+
+function isModelTypeURL(model: string): boolean {
+	return /^http(s?):/.test(model) || model.startsWith("/");
+}
+
+function getPipelineURL(model: string, pipeline: string): string {
+	return `${HF_INFERENCE_API_PIPELINE_BASE_URL}${pipeline}/${model}`;
+}
+
+function getDefaultModelTaskURL(model: string) {
+	return `${HF_INFERENCE_API_MODEL_BASE_URL}${model}`;
+}
```

Comment:
Those are one-line local functions used once, can you hardcode them directly in `makeRequestOptions`?